packages/fxa-settings/src/pages/Signup/SignupConfirmed/index.stories.tsx

```diff
@@ -16,12 +16,24 @@ export default {
 
 export const DefaultSignedIn = () => (
   <LocationProvider>
-    <SignupConfirmed isSignedIn />
+    <SignupConfirmed isSignedIn isSync={false} />
   </LocationProvider>
 );
 
 export const DefaultSignedOut = () => (
   <LocationProvider>
-    <SignupConfirmed isSignedIn={false} />
+    <SignupConfirmed isSignedIn={false} isSync={false} />
+  </LocationProvider>
+);
+
+export const IsSync = () => (
+  <LocationProvider>
+    <SignupConfirmed isSignedIn={false} isSync />
+  </LocationProvider>
+);
+
+export const IsNotSync = () => (
```

Comment:
Is this the same as `DefaultSignedOut` above?

Reply:
Ohhh yeah it is -- do you think it's better to have them both for the sake of being explicit, or better to rename so that it's clear that a single story applies to both cases?

Reply:
If the default is "not sync", then you might not need a story for "IsNotSync"?